tinydrive/configuration.md

```diff
@@ -83,6 +83,26 @@ tinymce.init({
 
 ## Configuring the Tiny Drive UI
 
+### tinydrive_skin
+
+This option allows you to specify the skin that {{site.cloudfilemanager}} should use. The default skin included with {{site.cloudfilemanager}} is named "oxide".
+
+**Type:** `String`
+
+**Default Value:** `oxide`
+
+**Possible Values:** `oxide`, `oxide-dark`
```

Comment:
Based on this, I expect that this option doesn't yet allow you to use custom or premium skins?

Reply:
That's correct, it's not possible to create custom skins for Tiny Drive and the extra skin we provide isn't an add-on.

Reply:
ok. just checking.